localization/english/religion/wc_magic_doctrines_l_english.yml

```diff
@@ -37,7 +37,7 @@
  doctrine_elemental_earth_magic_name:0 "Earth Magic"
 
  special_doctrine_elunic_gender_stratification_name:0 "Gender Stratification"
- special_doctrine_elunic_gender_stratification_desc:0 "For millennia, all priestesses were women and all druids were men, and to change it, tremendous societal change are required."
+ special_doctrine_elunic_gender_stratification_desc:0 "For millennia, all priestesses were women and all druids were men, and to change it would entail tremendous societal upheval."
```

Comment:
```suggestion
 special_doctrine_elunic_gender_stratification_desc:0 "For millennia, all priestesses were women and all druids were men, and to change it would entail tremendous societal upheaval."
```